modules/ROOT/pages/_partials/nav-app-dev.adoc

```diff
@@ -10,6 +10,7 @@
  *** xref:cache-scope.adoc[Cache Scope]
```

Comment:
If we're covering more than one component in this section, should we user "Cache Components," but then that could also require changes to cache-scope.adoc